src/components/SymbolModal.js

```diff
@@ -291,9 +295,14 @@ class SymbolModal extends Component {
   }
 
   render() {
-    const { enabled } = this.props;
+    const { enabled, selectedSource } = this.props;
+    let modalEnabled = false;
+
+    if (selectedSource && enabled) {
+      modalEnabled = true;
+    }
```

Comment:
we might be playing too much of a dance w/ enabled, perhaps a new method is in order:

* we dont need enabled prop
* we can consolidate logic

```js
isEnabled() {
  const {activeSearch, selectedSource} = this.props
  return activeSearch === "symbol" && selectedSource
}
```